content/docs/command-reference/exp/run.md

```diff
@@ -42,7 +46,11 @@ It's also possible to run special [checkpoint experiments] that log the
 execution progress (useful for deep learning ML). The `--rev` and `--reset`
 options have special uses for these.
 
-> 📖 See the [Running Experiments] guide for more details on all these features.
+<admon type="info">
+
+See the [Running Experiments] guide for more details on all these features.
+
+</admon>
```

Comment:
This book icon links are OK I think. We still use them. Or maybe we need a new `<admon icon="book">` ⏳

Reply:
https://github.com/iterative/gatsby-theme-iterative/pull/76